src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index].name();
     }
 }
```

Comment:
Please ensure that the class has an access modifier specified. In Java, it's a good practice to define the visibility of your classes explicitly.